site/content/docs/5.1/components/orange-footer.md

```diff
@@ -7,31 +7,89 @@ aliases: "/docs/5.1/components/orange-footer/"
 toc: true
 ---
 
-## Overview
+## How it works
 
-{{< callout warning >}}
-### Orange footer is not a component yet
+Orange footer is basically a `<footer>` tag with a `.bg-dark` class. It comes with several already built parts:
+- Title with content bar
+- Social media bar
+- Navigation bar
+- Service bar
+- **Mandatory** bar
```

Comment:
Put the mandatory bar first?

Reply:
As we discussed, added a sentence to make sure that the mandatory part should be used at the end of the component.

Reply:
As we discussed, added a sentence to make sure that the mandatory part should be used at the end of the component.